src/main/java/farmsystem/backend/domain/member/service/MemberService.java

```diff
@@ -0,0 +1,65 @@
+package farmsystem.backend.domain.member.service;
+
+import farmsystem.backend.domain.member.dto.request.MemberLoginRequest;
+import farmsystem.backend.domain.member.dto.request.MemberSignupRequest;
+import farmsystem.backend.domain.member.dto.response.MemberLoginResponse;
+import farmsystem.backend.domain.member.dto.response.MemberResponse;
+import farmsystem.backend.domain.member.entity.Member;
+import farmsystem.backend.domain.member.repository.MemberRepository;
+import farmsystem.backend.domain.profile.entity.Profile;
+import farmsystem.backend.domain.profile.entity.ProfileType;
+import farmsystem.backend.global.common.JwtUtil;
+import farmsystem.backend.global.error.ErrorCode;
+import farmsystem.backend.global.error.exception.ConflictException;
+import farmsystem.backend.global.error.exception.EntityNotFoundException;
+import farmsystem.backend.global.error.exception.InvalidValueException;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.time.LocalDate;
+
+@Service
+@RequiredArgsConstructor
+public class MemberService {
+    private final PasswordEncoder passwordEncoder;
+    private final MemberRepository memberRepository;
+    private final JwtUtil jwtUtil;
+
+    @Transactional
+    public MemberResponse signUp(MemberSignupRequest memberSignupRequest) {
+        if(memberRepository.existsByEmail(memberSignupRequest.email())){
+            throw new ConflictException(ErrorCode.CONFLICT);
```

Comment:
에러 코드 만든거 사용하면 됩니다!!
```suggestion
            throw new ConflictException(ErrorCode.MEMBER_EMAIL_EXIST);
```